shop3/explicit-stack-search/plan-tree-simple-print.lisp

```diff
@@ -0,0 +1,124 @@
+(in-package :plan-tree)
+
+(defparameter *output-form* nil)
+
+(defmethod print-node ((plan-tree-node top-node))
```

Comment:
Please add a `defgeneric` with docstring to explain what this is for and how to invoke it.  Do we call `print-node` directly, or is there an outer function to invoke?

When we get the docstrings, we should reference them in the manual (I can explain how this is done).